rfcs/0024-Non-consensus-node-protocol.md

```diff
@@ -0,0 +1,222 @@
+## Summary
+
+This RFC proposes a design for a protocol that allows users in low-capacity environments such as browsers, smartphones, and some desktops to verify the state of the network and perform operations such as retrieving account balance, sending payments etc with the same (similar) level of assurance as that of a full-node.
+
+The node that follows this protocol, is called a non-consensus node and can verify the state of the network without having to store the entire state. These nodes do not produce blocks but track enough information required to verify the state of the network. They rely on block producers and archive nodes for all other data and can easily validate them. The aim is to also reduce bandwidth usage by requiring only a subset of information from the network.
+
+## Requirements
+
+Coda daemon should by default run the non-consensus node and be able to switch to being consensus nodes
```

Comment:
switch dynamically, or at startup?

Reply:
A separate executable for non-consensus nodes for now. I'll update this.